### DISCUSSION TITLE
下载 DCE 5.0 社区版 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#975] [Announcements]
# 下载 DCE 5.0 社区版 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/download/free/dce5-installer-v0.4.0/

**Comment by san3Xian** [comment]
这安装包有点大呀T_T

**Reply by jzhupup** [reply]
好嘞，后续我们优化下呢
